src/pyOpenMS/CMakeLists.txt

```diff
@@ -341,13 +381,26 @@ add_custom_target(pyopenms_create_cpp
 	DEPENDS OpenMS SuperHirn
 	WORKING_DIRECTORY ${CMAKE_BINARY_DIR}/pyOpenMS )
 
-add_custom_target(pyopenms
-	COMMAND ${PYTHON_EXECUTABLE} setup.py build_ext
-	COMMAND ${PYTHON_EXECUTABLE} setup.py bdist_egg
-	COMMAND ${PYTHON_EXECUTABLE} setup.py bdist --format=zip
-	COMMAND ${PYTHON_EXECUTABLE} setup.py build_ext --inplace
-	DEPENDS pyopenms_create_cpp
-	WORKING_DIRECTORY ${CMAKE_BINARY_DIR}/pyOpenMS )
+IF(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+    add_custom_target(pyopenms
+        COMMAND ${PYTHON_EXECUTABLE} setup.py build_ext
+        COMMAND fix_pyopenms_dependencies_on_mac.sh
```

Comment:
ld: warning: directory not found for option '-L/pyopenms/bin/Release'
ld: warning: directory not found for option '-L/pyopenms/Release'
ld: warning: directory not found for option '-L/pyopenms/contrib/lib'
/bin/sh: fix_pyopenms_dependencies_on_mac.sh: command not found
make[3]: *** [src/pyOpenMS/CMakeFiles/pyopenms] Error 127